charts/orchestrator-k8s/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: orchestrator-k8s
+description: |
+  Helm chart to deploy the Orchestrator solution suite on Kubernetes, including Janus IDP backstage, SonataFlow Operator, Knative Eventing and Knative Serving.
+type: application
+version: 0.2.0
+appVersion: "0.0.1"
+
+dependencies:
+  - name: backstage
+    repository: https://janus-idp.github.io/helm-backstage
+    version: "2.11.4"
+  - name: postgresql-persistent
+    repository: https://sclorg.github.io/helm-charts
+    version: "0.0.3"
+    condition: postgresql-persistent.enabled
+  - name: workflows
```

Comment:
Perhaps we can target only the `greeting` workflow from that repository and let other workflows be installed by the user since they might require more steps (at least m2k).
```
→ helm search  repo orchestrator-workflows 
NAME                            	CHART VERSION	APP VERSION	DESCRIPTION                                       
orchestrator-workflows/greeting 	0.4.5        	1.16.0     	A Helm chart for the greeting serverless workflow 
orchestrator-workflows/move2kube	0.2.21       	1.16.0     	A Helm chart to deploy the move2kube workflow.    
orchestrator-workflows/mta      	0.2.21       	1.16.0     	A Helm chart for MTA serverless workflow          
orchestrator-workflows/workflows	0.2.26       	1.16.0     	A Helm chart for serverless workflows. This cha...
```